stingray/tests/test_lightcurve.py

```diff
@@ -27,6 +29,17 @@
 except ImportError:
     _HAS_LIGHTKURVE = False
 
+try:
+    import astropy.timeseries
+    from astropy.timeseries import TimeSeries
```

Comment:
Is this check because we're supporting old versions of astropy that don't have `TimeSeries` yet? Because if it is, I'd like to point out that the changes above in `events` and `Lightcurve` will definitely break. in this new implementation with old versions of `astropy`. So I think we either need to make it optional everywhere, or just require it (I'm fine with just requiring it)